wherehows-backend/jobs/README.md

```diff
@@ -13,6 +13,13 @@ job.cmd.params | Optional extra command line arguments to pass to the job
 job.cmd.disabled | If defined, regardless of it value, the job will not be run
 job.ref.id | A unique reference ID for the job (To be removed in the future)
 
+For ETL specific job configurations, we can define the same key with multiple values in different lines.
```

Comment:
Please also mention that these values will be combined into a comma-separated list

Reply:
Updated it now.